src/node/share_manager.h

```diff
@@ -96,6 +110,11 @@ namespace ccf
       return ret;
     }
 
+    std::vector<uint8_t> get_full_share_serialised() const
```

Comment:
This is a problem because it will leave a vector with the full share on the stack, we need to call with a member id instead, and encrypt (same as wrap()) so as to only return encrypted info.

I think it would be better to modify wrap() to return a WrappedShare, with a map of member_id to wrapped_shared, all encrypted.

Reply:
@achamayou I'm not clear how exactly to make this change. Perhaps you could show it how?